docs/reference/advanced_configuration.rst

```diff
@@ -113,6 +113,11 @@ Full configuration options:
                     key1: value1 #any amount of metas(sent as x-amz-meta-key1 = value1)
                 async: false
 
+            azure:
+                containerName:
+                connectionString:
```

Comment:
Except accessKey and secretKey, I feel like we're using snake case everywhere.
I think those "key" were a mistake and we should stick with snake case, WDYT ?

So container_name and connection_string (and create_container)